diff --git a/src/Makefile b/src/Makefile
index 5df880b..772959c 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -2,37 +2,38 @@ UNAME := $(shell uname)
 
 ifeq ($(UNAME), Linux)
 # Linux
-	CC = g++
+#	CC = g++
 	SHARED_FLAG = -shared
 else
 	ifeq ($(UNAME), Darwin)
 # OS X
-		CC = g++
+#		CC = g++
 		SHARED_FLAG = -dynamiclib
 	else
-		CC = g++
+#		CC = g++
 	endif
 endif
 
 CXXFLAGS = -Wall -fopenmp -O3 -march=native -DNDEBUG -std=c++11
 
-INCLUDES = -I${EIGEN_DIR} -I${BOOST_DIR} -I/usr/include/eigen3
+#INCLUDES = -I${EIGEN_DIR} -I${BOOST_DIR} -I/usr/include/eigen3
+INCLUDES = -I${EIGEN_ROOT}/include/eigen3 -I${BOOST_ROOT}/include
 LDFLAGS = -lpthread
 
 all: Grid.o Grid_ctypes.o FDTD.o Grid.cpp Grid.hpp Grid_ctypes.cpp Grid_ctypes.hpp fdtd.cpp fdtd.hpp
-	${CC} ${SHARED_FLAG} -fopenmp -fPIC -o Grid.so Grid.o Grid_ctypes.o ${LDFLAGS}
-	${CC} ${SHARED_FLAG} -fopenmp -fPIC -o FDTD.so FDTD.o ${LDFLAGS}
+	${CXX} ${SHARED_FLAG} -fopenmp -fPIC -o Grid.so Grid.o Grid_ctypes.o ${LDFLAGS}
+	${CXX} ${SHARED_FLAG} -fopenmp -fPIC -o FDTD.so FDTD.o ${LDFLAGS}
 	cp Grid.so ../emopt
 	cp FDTD.so ../emopt
 
 Grid.o: Grid.cpp Grid.hpp
-	${CC} -c -fPIC Grid.cpp ${CXXFLAGS} -o Grid.o ${INCLUDES}
+	${CXX} -c -fPIC Grid.cpp ${CXXFLAGS} -o Grid.o ${INCLUDES}
 
 Grid_ctypes.o: Grid.cpp Grid.hpp Grid_ctypes.cpp Grid_ctypes.hpp
-	${CC} -c -fPIC Grid_ctypes.cpp ${CXXFLAGS} -o Grid_ctypes.o ${INCLUDES}
+	${CXX} -c -fPIC Grid_ctypes.cpp ${CXXFLAGS} -o Grid_ctypes.o ${INCLUDES}
 
 FDTD.o: fdtd.cpp fdtd.hpp
-	${CC} -c -fPIC fdtd.cpp ${CXXFLAGS} -o FDTD.o
+	${CXX} -c -fPIC fdtd.cpp ${CXXFLAGS} -o FDTD.o
 
 clean:
 	rm -f ./*.o
